browser/ui/views/bookmarks/bookmark_bar_instructions_view.cc

```diff
@@ -145,6 +145,6 @@ void BookmarkBarInstructionsView::UpdateColors() {
       GetNativeTheme()->GetSystemColor(ui::NativeTheme::kColorId_LinkEnabled);
   bool link_has_contrast = color_utils::GetContrastRatio(link_color, bg) >=
                            color_utils::kMinimumReadableContrastRatio;
-  import_link_->SetUnderline(!link_has_contrast);
+  //import_link_->SetUnderline(!link_has_contrast);
```

Comment:
@simonhong is the underline style required here or can we just remove?

Reply:
I think we can remove it.